selfdrive/car/car_helpers.py

```diff
@@ -91,7 +91,7 @@ def fingerprint(logcan, sendcan, has_relay):
     fw_candidates, car_fw = set(), []
 
   cloudlog.warning("VIN %s", vin)
-  Params().put("CarVin", vin)
+  put_nonblocking("CarVin", vin)
```

Comment:
We can't use the asynchronous version here, since boardd is waiting for the vin to be set and change the safety mode.